Firmware/odrive-interface.yaml

```diff
@@ -656,6 +694,12 @@ valuetypes:
       Homing:
         doc: run axis homing function
 
+  ODrive.ThermistorCurrentLimiter.Error:
+    values:
```

Comment:
For this enum the `flags` key should be used instead of `values` to make the interface generator chose numbers that are powers of two (see for example `ODrive.Motor.Error`). The `nullflag` key can then be used to specify the name of the "no flag is set"-state.

Reply:
Fixed.